Project.toml

```diff
@@ -33,6 +33,7 @@ CUDA = "052768ef-5323-5732-b1bb-66c8b64840ba"
 QuantumToolboxCUDAExt = "CUDA"
 
 [compat]
+Aqua = "0.8"
 ArrayInterface = "6, 7"
 CUDA = "5"
```

Comment:
Should we remove `CUDA` here?

Reply:
We can't
we still have weak dependent on `CUDA`